openapi.yaml

```diff
@@ -0,0 +1,189 @@
+openapi: 3.0.3
+info:
+  title: genny
+  description: A RESTful API for generating dummy data against a schema.
+  version: 1.0.11
+tags:
+  - name: Data
+    description: Data Routes
+  - name: Schema
+    description: Schema Routes
+paths:
+  /data:
+    post:
+      tags:
+        - Data
+      summary: Generate Ephemeral
+      operationId: GenerateEphemeral
+      requestBody:
+        required: true
+        content:
+          application/json:
+            schema:
+              type: object
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                type: object
+    get:
+      tags:
+        - Data
+      summary: Generate Objects
+      operationId: GenerateObjects
+      parameters:
+        - name: schema
+          in: query
+          schema:
+            type: string
+        - name: count
+          in: query
+          schema:
+            type: integer
+            format: int32
+      responses:
+        '200':
+          description: Successful operation
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  type: object
+  /data/values:
+    get:
+      tags:
+        - Data
+      summary: Generate Values
+      operationId: GenerateValues
+      parameters:
+        - name: formula
+          in: query
+          required: true
+          schema:
+            type: string
+        - name: count
+          in: query
+          required: true
+          schema:
+            type: integer
+            format: int32
```

Comment:
Is the type of schema really an int? This query parameter can accept either an ObjectId (hex string) or a slug (arbitrary string)